controller/controller.go

```diff
@@ -395,6 +395,7 @@ func ScaleUp(
 		return err
 	}
 	if existPendingPods {
+		// TODO(lhahn): Important.
```

Comment:
Will you leave this TODO here?

If yes: does it make sense to describe more specifically what should be done here?

Reply:
I'll remove them.